mountpoint-s3/src/fuse.rs

```diff
@@ -223,6 +223,17 @@ where
         }
     }
 
+    #[instrument(level="debug", skip_all, fields(req=_req.unique(), parent=parent, name=?name))]
+    fn mkdir(&self, _req: &Request<'_>, parent: u64, name: &OsStr, mode: u32, umask: u32, reply: ReplyEntry) {
```

Comment:
Why mkdir does not have rdev as one of the arguments unlike mknod? I can see rdev is used to query/set image root device, RAM disk size, or video mode. I dont find this specific to file or directory.